MacPass/MPPasswordInputController.h

```diff
@@ -24,7 +24,12 @@
 
 @class KPKCompositeKey;
 
-@interface MPPasswordInputController : MPViewController
+@interface MPPasswordInputController : MPViewController <NSTouchBarDelegate> {
+  @private NSTouchBarCustomizationIdentifier touchBarIdentifier;
```

Comment:
Since you set them as private, no need to vent them in the header. Additionally you could just use static NSStrings in the .m file.